components/html/modules/content-blank/_all.html

```diff
@@ -0,0 +1,4 @@
+@@include('../navigation/_breadcrumbs.html')
```

Comment:
We should probably remove the breadcrumb HTML since not all of our templates use it in the first place.